### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by osdevisnot**
This PR updates dependencies to their latest version.

### CODE REVIEW

**Comment by stonebk** [comment]
Can you update the commit names to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines?

File: src/bin/commands/new.js [hunk 1]
```diff
@@ -14,17 +14,17 @@ const inquirer = require('inquirer');
 const STABLE_VERSIONS = {
```
Comment (stonebk) [inline src/bin/commands/new.js]: Did you validate these changes are stable?

File: package.json [hunk 2]
```diff
@@ -38,35 +38,35 @@
...
-    "eslint-plugin-zillow": "^3.4.1",
-    "husky": "^3.0.8",
-    "react": "^16.10.2",
-    "react-dom": "^16.10.2",
-    "standard-version": "^6.0.1",
+    "eslint-plugin-zillow": "^3.5.1",
+    "husky": "^3.1.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "standard-version": "^7.0.1",
```
Comment (stonebk) [inline package.json]: I'm wary about upgrading this -- the last few times I have tried it produced unexpected results when creating the CHANGELOG.

File: src/bin/commands/new.js (hunk 1 above)
Comment (osdevisnot) [reply src/bin/commands/new.js]: Running few more validations, will confirm when everything works perfect. please wait for ack.

**Comment by osdevisnot** [comment]
> Can you update the commit names to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines?

Added conventional commit prefixes.

File: package.json (hunk 2 above)
Comment (osdevisnot) [reply package.json]: Do we have pointers on specific problems? Would it be worth filing issues to `standard-version` for specific behaviour?

Comment (stonebk) [reply package.json]: It's been a while, but maybe the issue was just this:
https://github.com/conventional-changelog/standard-version/issues/424

**Comment by stonebk** [comment]
> > Can you update the commit names to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines?
> 
> Added conventional commit prefixes.

The commits should be of the form:

```
chore: this is the message
```

...as opposed to...

```
(chore) this is the message
```

File: src/bin/commands/new.js (hunk 1 above)
Comment (osdevisnot) [reply src/bin/commands/new.js]: downgraded `standard-version` for now. every other update here works correctly.

Comment (stonebk) [reply src/bin/commands/new.js]: Published `create-react-styleguide@prerelease` for final testing

Comment (stonebk) [reply src/bin/commands/new.js]: nitpick: I'd squash the "revert standard-version to previous release" commit into your first commit just for a clean commit history

**Comment by stonebk** [comment]
Found an issue on the prerelease version:
Screen Shot 2019-11-20 at 8 16 00 AM

I think this is why I haven't been upgrading react-styleguidist.

**Comment by stonebk** [comment]
We should probably document the validation steps:

1. Publish a prerelease version: `npm run release -- --prerelease`
2. Create a new project to validate: `npx create-react-styleguide@prerelease new crs-verify`
3. Update project to use the prerelease version:
    * `cd crs-verify`
    * `npm i creact-react-styleguide@prerelease`
4. Validate the living styleguide
    * `npm start`
    * Go to http://localhost:6060/
5. Validate the build script: `npm run build`
6. Validate the testing scripts
    * `npm run test`
    * `npm run eslint`

**Comment by stonebk** [comment]
Closing in favor of https://github.com/zillow/create-react-styleguide/pull/30

**Pull request closed without merging**